docs/parallelism.rst

```diff
@@ -31,12 +31,12 @@ An example of use to extract features on a :class:`lhotse.CutSet`:
 .. code-block::
 
     from concurrent.futures import ProcessPoolExecutor
-    from lhotse import CutSet, Fbank, LilcomFilesWriter
+    from lhotse import CutSet, Fbank, ChunkedLilcomHdf5Writer
```

Comment:
is there a reason you prefer the chunked HDF5 version to the default `LilcomChunkyWriter`?

Reply:
No, I did not realize that there is a `LilcomChunkyWriter`.
Just reverted to `LilcomChunkyWriter`.